core/templates/dev/head/pages/review-test-page/review-test-engine.service.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
Missing frontend tests?

Please make sure to write tests for new code. Otherwise the reviewer will just ask for them, and that delays the PR.

Reply:
Done.